firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/ImagenImageFormat.kt

```diff
@@ -1,13 +1,43 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.google.firebase.vertexai.type
 
+/**
+ * Represents the format an image should be returned in.
+ * @param mimeType A string (like `"image/jpeg"`) specifying the encoding MIME type of the image.
+ * @param compressionQuality an int (1-100) representing the quality of the image; a lower number
+ * means the image is permitted to be lower quality to reduce size. This parameter is not relevant
+ * for every MIME type.
```

Comment:
Do you know if this is documented somewhere?  i.e., which MIME types this is relevant to?

Reply:
Its documented by the method signitures, for example in the png() method there is no compression quality param because its not relevant there.